### PR TITLE
feat: 播放器中使用 Layer 定位

### PR DESCRIPTION
**Pull request by TaraLoveCats**
# Pull Request Template

## Description

Player 中使用 Layer 层定位传入的组件

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] 本地测试

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/griffith/src/components/Player.tsx [hunk 1]
```diff
@@ -691,6 +694,7 @@ const InnerPlayer: React.FC<InnerPlayerProps> = ({
       )}
       {controlsOverlay}
       <ActionToastOutlet />
+      <Layer>{PositioningComponent}</Layer>
```
Comment (ambar) [inline packages/griffith/src/components/Player.tsx:697]: - 这是一个 jsx 元素就不应该用 PascalCase。
- 可以直接叫 layer？
- 这样可能好一些 {layer && ..}

Comment (TaraLoveCats) [reply packages/griffith/src/components/Player.tsx:697]: 写错了，传进来的是一个组件不是 jsx 是 ReactNode

Comment (TaraLoveCats) [reply packages/griffith/src/components/Player.tsx:697]: 命名有建议吗？

Comment (ambar) [reply packages/griffith/src/components/Player.tsx:697]: 看上面，传 jsx element 就行。

**Pull request merged**